.env.test

```diff
@@ -1,5 +1,6 @@
 # define your env variables for the test env here
 KERNEL_CLASS='App\Kernel'
 APP_SECRET='s$cretf0rt3st'
+APP_DEBUG=true
```

Comment:
This can maybe be removed when the found a solution for the cache warmup: https://github.com/symfony/recipes/pull/530/files

Reply:
I don't really get why this is necessary :thinking:

Reply:
@danrot APP_DEBUG=true is necessary in test environment to avoid cleaning the cache manually everytime you change something in the container or routes.